pyppeteer/browser_fetcher.py

```diff
@@ -25,10 +25,9 @@
 
 import certifi
 import urllib3
-from tqdm import tqdm
-
 from pyppeteer import __chromium_revision__, __pyppeteer_home__
 from pyppeteer.models import Platform, RevisionInfo
+from tqdm import tqdm
```

Comment:
[It's a generally accepted convention that third party imports come before self-package imports](https://www.python.org/dev/peps/pep-0008/#imports). pre-commit hooks are available in the repo to fix this sort of thing automatically.

Reply:
Sorry, overlooked this. the import was automatically moved by isort, which requires a package to be installed to classify it as a third-party.